dags/kpi_forecasting.py

```diff
@@ -0,0 +1,93 @@
+"""
+See [kpi-forecasting in the docker-etl repository]
+(https://github.com/mozilla/docker-etl/blob/main/jobs/kpi-forecasting).
+
+This DAG is low priority.
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from utils.gcp import gke_command
+from operators.task_sensor import ExternalTaskCompletedSensor
+from utils.tags import Tag
+
+default_args = {
+    "owner": "pmcmanis@mozilla.com",
+    "email": ["pmcmanis@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2022, 3, 28),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+tags = [Tag.ImpactTier.tier_3]
+
+with DAG("kpi_forecasting", default_args=default_args, schedule_interval="0 4 * * SAT", doc_md=__doc__, tags=tags,) as dag:
+    dataset_yamls = ["yaml/mobile.yaml", "yaml/desktop.yaml"]
+
+    kpi_forecasting = gke_command(
+        task_id="kpi_forecasting",
+        command=[
+            "python", "kpi-forecasting/kpi-forecasting.py",
+            "-c",
+        ] + dataset_yamls,
```

Comment:
Can we split these out to separate tasks for desktop and mobile? I would expect we may want to be able to rerun separately if we fix upstream issues that are specific to desktop vs. mobile.

Reply:
yes, that makes sense